src/app/shared/entry/descriptor-language.service.ts

```diff
@@ -46,6 +59,106 @@ export class DescriptorLanguageService {
     });
   }
 
+  toolDescriptorTypeEnumToWeirdCheckerRegisterString(descriptorType: ToolDescriptor.TypeEnum): 'cwl' | 'wdl' | null {
```

Comment:
this in particular could use [JSDoc](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#angular)